src/modelmatrix.jl

```diff
@@ -1,6 +1,22 @@
 
 typealias AbstractFloatMatrix{T<:AbstractFloat} AbstractMatrix{T}
 
+"""
+Convert a `ModelFrame` into a numeric matrix suitable for modeling
```

Comment:
If you document the constructors, give the signatures first like any other function? I'm not sure what are the conventions for this.